src/Uno.UI/UI/Xaml/Controls/DatePicker/DatePicker.md

```diff
@@ -14,19 +14,24 @@ There might be differences in the date picker on different platform since it wra
 
 #### Android
 
-Native date picker is wrapped in the flyout.
-DatePicker flyout appear centered to screen.
+- Native date picker is wrapped in the flyout.
+- DatePicker flyout appear centered to screen.
 
 #### iOS
-Native date picker is wrapped in the flyout.
-Set 'FlyoutPlacement' property to change flyout docking placement
-Default 'FlyoutPlacement' is 'Full' and will dock of the flyout at the bottom of the screen
-
+- Native date picker is wrapped in the flyout.
+- Set '**FlyoutPlacement**' property to change flyout docking placement
+- Default '**FlyoutPlacement**' is '**Full**' and will dock of the flyout at the bottom of the screen
 
 If you want to show a dimmed overlay underneath the picker, set the `DatePicker.LightDismissOverlayMode` property to `On`.
 
 If you wish to customise the overlay color, add the following to your top-level `App.Resources`:
 ```xaml
-		<SolidColorBrush x:Key="DatePickerLightDismissOverlayBackground"
-						 Color="Pink" />
+<SolidColorBrush x:Key="DatePickerLightDismissOverlayBackground"
+		Color="Pink" />
+```
+
+Since **iOS14** the native DatePicker changed the way it's presented. To keep consistency, the previous style (< iOS13) was defined as the default for this Control.
+If you want iOS14 devices to use the brand new looking **Picker** you just need to add the following at your top-level `App.Resources`:
```

Comment:
This is not how we normally add feature flags - you should use the [FeatureConfiguration class](https://github.com/unoplatform/uno/blob/master/src/Uno.UI/FeatureConfiguration.cs)

Reply:
@davidjohnoliver Updated to use the `FeatureConfiguration` class. Thanks for pointing me out to this class.

Reply:
No worries :) We should add it to the contributor docs section (maybe here - https://github.com/unoplatform/uno/blob/master/doc/articles/contributing/guidelines/code-style.md/#patterns )

Actually, it's worth a whole article on how we go about it when adding parts to the public API that aren't from WinUI...